testing/data-generator/src/main/java/com/simprints/feature/datagenerator/enrollmentrecords/InsertEnrollmentRecordsUseCase.kt

```diff
@@ -134,16 +134,46 @@ internal class InsertEnrollmentRecordsUseCase @Inject constructor(
     }
 
     private fun String.toFingerIdentifier() = when (this.uppercase()) {
-        "LEFT_THUMB" -> SampleIdentifier.LEFT_THUMB
-        "LEFT_INDEX_FINGER" -> SampleIdentifier.LEFT_INDEX_FINGER
-        "LEFT_3RD_FINGER" -> SampleIdentifier.LEFT_3RD_FINGER
-        "LEFT_4TH_FINGER" -> SampleIdentifier.LEFT_4TH_FINGER
-        "LEFT_5TH_FINGER" -> SampleIdentifier.LEFT_5TH_FINGER
-        "RIGHT_THUMB" -> SampleIdentifier.RIGHT_THUMB
-        "RIGHT_INDEX_FINGER" -> SampleIdentifier.RIGHT_INDEX_FINGER
-        "RIGHT_3RD_FINGER" -> SampleIdentifier.RIGHT_3RD_FINGER
-        "RIGHT_4TH_FINGER" -> SampleIdentifier.RIGHT_4TH_FINGER
-        "RIGHT_5TH_FINGER" -> SampleIdentifier.RIGHT_5TH_FINGER
+        "LEFT_THUMB" -> {
```

Comment:
If you insist on changing this file, I would rather remove the brackets from the else branch.